packages/server/lib/errors.js

```diff
@@ -600,12 +600,12 @@ const getMsgByType = function (type, ...args) {
         The \`setupNodeEvents\` method does not support \`dev-server:start\`, use \`devServer\` instead:
 
         \`\`\`
-        devServer (cypressConfig, devServerConfig) {
+        devServer (cypressDevServerConfig, devServerConfig) {
           // configure plugins here
```

Comment:
Sorry I missed this before - but can you change this comment to `// start dev server here` and also add it to the `devServer` example above?